packages/flutter/lib/src/material/theme.dart

```diff
@@ -32,6 +32,14 @@ const Duration kThemeAnimationDuration = Duration(milliseconds: 200);
 /// The [Theme] widget implies an [IconTheme] widget, set to the value of the
 /// [ThemeData.iconTheme] of the [data] for the [Theme].
 ///
+/// To interact seamlessly with descendant Cupertino widgets, the [Theme] widget
+/// implies a [CupertinoTheme] widget with a [CupertinoThemeData] inherited from
```

Comment:
What do you mean by "implies" here?

Reply:
I was trying to say "internally creates a `CupertinoTheme` for its descendants" without saying it. Maybe I should just say "the [Theme] widget provides a [CupertinoTheme] for its descendants with a [CupertinoThemeData]....".

Reply:
Yeah I think that's better. I think "implies" could be interpreted to mean that CupertinoTheme must be present elsewhere or something like that.